LayerSelector.js

```diff
@@ -55,7 +55,7 @@ define([
         _getVisibleLayersAttr: function baseLayers() {
             var layers = this.get('managedLayers');
 
-            var visibleLayerWidgets = this.baseLayerWidgets.filter(function findVisibleBaseLayers(widget) {
+            var visibleLayerWidgets = array.filter(this.baseLayerWidgets, function findVisibleBaseLayers(widget) {
```

Comment:
We should put the the supported browsers in the README. What ones are you targeting?